scripts/secureboot-certs

```diff
@@ -446,10 +446,17 @@ if __name__ == "__main__":
     install_parser = action_parsers.add_parser(
         Actions.INSTALL,
         help="Install UEFI certificates to every host in the pool.",
-        description="Install UEFI certificates to every host in the pool.",
+        description=(
+              "Install UEFI certificates to every host in the pool.\n\n"
+              "If no arguments are passed to '{} {}', then the default PK, KEK, "
+              "and db, and the latest dbx will be installed by default."
```

Comment:
There's "default" twice in the sentence. Would it be better without the second occurrence?

Reply:
Yes, I think so.